setup.py

```diff
@@ -30,40 +30,44 @@ def run_tests(self):
     version="1.5.0",
     author="Bryant Moscon",
     author_email="bmoscon@gmail.com",
-    description=("Cryptocurrency feed handler and synthetic NBBO feed"),
-    long_description=ld,
-    long_description_content_type='text/x-rst',
+    description="Cryptocurrency feed handler and synthetic NBBO feed",
+    long_description=readme + '\n\n' + changes,
+    long_description_content_type='text/markdown',
     license="XFree86",
     keywords=["cryptocurrency", "bitcoin", "btc", "feed handler", "market feed", "market data"],
     url="https://github.com/bmoscon/cryptofeed",
-    packages=find_packages(exclude=['tests']),
+    packages=find_packages(exclude=['tests*']),
     package_data={'': ['rest/config.yaml']},
     cmdclass={'test': Test},
+    python_requires='>=3.7',
     classifiers=[
+        "Intended Audience :: Developers",
         "Development Status :: 4 - Beta",
+        "Programming Language :: Python :: 3 :: Only",
         "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8"
+        "Programming Language :: Python :: 3.8",
+        "Framework :: AsyncIO",
     ],
     tests_require=["pytest"],
     install_requires=[
         "requests>=2.18.4",
         "websockets>=7.0",
         "sortedcontainers>=1.5.9",
         "pandas",
-        "pyyaml",
         "aiohttp==3.6.2",
-        "aiodns",
```

Comment:
i get enough issues opened already from people wanting as much performance as possible that not requiring aiodns and cchardet seems like a step backwards

Reply:
OK I revert that change :+1: